users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class User(models.Model):
+    name        = models.CharField(max_length=45)
+    email       = models.EmailField()
+    password    = models.CharField(max_length=200)
+    sms_agree   = models.BooleanField(default=False)
```

Comment:
boolean 값을 갖는 경우 `is_activated`와 같은 형식으로 컬럼명을 정합니다~

```suggestion
    is_sms_agreed   = models.BooleanField(default=False)
```